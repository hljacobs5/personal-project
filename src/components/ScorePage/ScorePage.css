.score {
	font-size: 10rem;
}