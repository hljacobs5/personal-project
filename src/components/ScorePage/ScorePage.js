import React from 'react';
import './ScorePage.css';
import { connect } from 'react-redux';

const ScorePage = (props) => {
	return (
		<div>
			<h1>{props.score}/10</h1>
		</div>
	)
}

export const mapStateToProps = (state) => ({
	score: state.score
})

export default connect(mapStateToProps)(ScorePage);