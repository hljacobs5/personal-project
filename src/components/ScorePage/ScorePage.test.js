import React from 'react';
import { shallow, mount } from 'enzyme';
import { ScorePage } from './ScorePage';

describe('ScorePage', () => {
	let wrapper;

	it('should exist', () => {
		expect(wrapper).toBeDefined();
	})

	it('should render like snapshot', () => {
		expect(wrapper).toMatchSnapshot();
	})
})