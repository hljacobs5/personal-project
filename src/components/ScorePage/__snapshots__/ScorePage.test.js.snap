// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScorePage should render like snapshot 1`] = `undefined`;
